libnetdata/socket/security.h

```diff
@@ -18,7 +18,7 @@
 #define NETDATA_SSL_VALID_CERTIFICATE 128  //Accepts invalid certificate
 struct netdata_ssl{
     SSL *conn; //SSL connection
-    int flags;
+    uint32_t flags;
```

Comment:
This change doesn't belong here.

Reply:
I will create a new PR for this, while I was compiling with Clion it showed some warnings due clang compiler and I changed here.

Reply:
Next pull will fix this!